analyses/new-feature-template.qmd

```diff
@@ -0,0 +1,1549 @@
+#       key = "performance"
+#     ),
+#     list(
+#       s3 = paste0("s3://ccao-model-results-us-east-1/assessment_pin/year=", params$comparison_run_id_year, "/run_id=", params$comparison_run_id, "/"),
+#       key = "assessment_pin"
+#     ),
+#     list(
+#       s3 = paste0("s3://ccao-model-results-us-east-1/shap/year=", params$comparison_run_id_year, "/run_id=", params$comparison_run_id, "/"),
+#       key = "shap"
+#     ),
+#     list(
+#       s3 = paste0("s3://ccao-model-results-us-east-1/metadata/year=", params$comparison_run_id_year, "/", params$comparison_run_id, ".parquet"),
+#       key = "metadata"
+#     )
+#   )
+# )
+#
+# data_comparison <- fetch_analyses(params$comparison_run_id, params$comparison_run_id_year, analyses_paths)
+#
+# metadata_comparison <- data_comparison$metadata
+# model_performance_assessment_comparison <- data_comparison$performance
+# shap_comparison <- data_comparison$shap
+#
+# assessment_pin_comparison <- data_comparison$assessment_pin %>%
+#   select(meta_pin, pred_pin_final_fmv, sale_ratio_study_price, meta_nbhd_code, meta_triad_code, pred_pin_initial_fmv, meta_township_code)
+```
+
+```{r}
+# write_parquet(assessment_data, "assessment_data.parquet")
+# write_parquet(assessment_pin, "assessment_pin.parquet")
+# write_parquet(assessment_card, "assessment_card.parquet")
+# write_parquet(performance, "performance.parquet")
+# write_parquet(shap, "shap.parquet")
+# write_parquet(metadata, "metadata.parquet")
+# write_parquet(assessment_pin_comparison, "assessment_pin_comparison.parquet")
+# write_parquet(model_performance_assessment_comparison, "model_performance_assessment_comparison.parquet")
+# write_parquet(shap_comparison, "shap_comparison.parquet")
+# write_parquet(metadata_comparison, "metadata_comparison.parquet")
+```
+
+```{r}
+assessment_data <- read_parquet("assessment_data.parquet")
+assessment_pin <- read_parquet("assessment_pin.parquet")
+assessment_card <- read_parquet("assessment_card.parquet")
+performance <- read_parquet("performance.parquet")
+shap <- read_parquet("shap.parquet")
+metadata <- read_parquet("metadata.parquet")
+assessment_pin_comparison <- read_parquet("assessment_pin_comparison.parquet")
+model_performance_assessment_comparison <- read_parquet("model_performance_assessment_comparison.parquet")
+shap_comparison <- read_parquet("shap_comparison.parquet")
+metadata_comparison <- read_parquet("metadata_comparison.parquet")
+```
+
+
+
+```{r data_processing}
+assessment_data_small <- assessment_data %>%
+  select(meta_pin, meta_card_num, meta_nbhd_code, loc_longitude, loc_latitude, meta_township_name, !!sym(params$added_feature))
+
+# Create a card level dataset
+
+working_data_card <- shap %>%
+  select(meta_pin, meta_card_num, pred_card_shap_baseline_fmv, !!sym(params$added_feature)) %>%
+  rename(!!params$added_feature_shap := !!sym(params$added_feature)) %>%
```

Comment:
I used !!sym throughout but it's really unweildy. Do we want it just saved as an object?